bus/EasyCaching.Bus.Redis/Configurations/RedisSubscriberProvider.cs

```diff
@@ -48,6 +48,9 @@ public ISubscriber GetSubscriber()
         /// <returns>The connection multiplexer.</returns>
         private ConnectionMultiplexer CreateConnectionMultiplexer()
         {
+            if (_options.ConfigurationOptions != null)
+                return ConnectionMultiplexer.Connect(_options.ConfigurationOptions.ToString());
```

Comment:
![image](https://user-images.githubusercontent.com/8394988/216774386-b6075431-08d0-4156-85be-90a6ea176f40.png)

Should we use this `Connect` method?